spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -181,6 +181,31 @@
       end
     end
 
+    context 'an invalid selection is submitted' do
+      # (This should only be possible if someone alters the form)
```

Comment:
How is this different from the form submission error case that's handled in the spec starting on line 266? Could those two specs be merged into one? I think that might be cleaner?

Reply:
Good catch.

My thinking at the time was that 266 tests when _no_ selection is made, while this test is when an _invalid_ selection is made, which wasn't previously handled.

But "invalid" is ambiguous wording used in both tests, and they should result in the same behavior.

Now the question is, do I make this a mini shared example or just duplicate it? 🤔

Reply:
OK yeah, we have specific behavior expectations for no selection vs. some selection so now that you say I think we should maybe keep them separate. Mini shared example is my preference, since it's what we'd end up doing if this spec gets larger anyway, right?